src/algorithm/geometry.hpp

```diff
@@ -93,6 +93,21 @@ namespace se3
                                 const GeometryModel & geomModel,
                                 GeometryData &        geomData);
 #endif // WITH_HPP_FCL
+
+  /// Append geomModel2 to geomModel1
+  ///
+  /// The steps for appending are:
+  /// \li add GeometryObject of geomModel2 to geomModel1,
+  /// \li add the collision pairs of geomModel2 into geomModel1 (indexes are updated)
+  /// \li add all the collision pairs between geometry objects of geomModel1 and geomModel2.
+  /// \li update the inner objects of geomModel1 with the inner objects of geomModel2
+  /// \li update the outer objects (see TODO)
+  ///
+  /// \warning Radius should be recomputed.
```

Comment:
Maybe all radius of joints whose body list has been increased should be set to nan? at least in debug mode.